cmake/configs/default.cmake

```diff
@@ -95,7 +95,7 @@ else()
   hunter_config(OpenSSL VERSION 1.1.0f)
 endif()
 hunter_config(PNG VERSION 1.6.26-p1)
-hunter_config(PocoCpp VERSION 1.7.6-p0)
+hunter_config(PocoCpp VERSION 1.7.6-p07)
```

Comment:
I tried remove dependencies, but it is failed.
https://travis-ci.org/rusdevops/hunter/jobs/258286597#L6682-L6696